### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by AnatoliiMarushchak**
- [DEMO LINK](https://AnatoliiMarushchak.github.io/layout_stars/)
- [TEST REPORT LINK](https://AnatoliiMarushchak.github.io/layout_stars/report/html_report/)

> * [x]  Yellow stars are added with container modifier + pseudo-selector (NO extra classes)
> * [x]  Each BEM block has its own separate file
> * [x]  All `Typical Mistakes` from `BEM` lesson theory are checked.
> * [x]  Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,37 @@
...
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url("./images/star-active.svg");
+}
```
Comment (volodymyr-soltys97) [inline src/style.css:37]: These selectors must be separated by a comma.

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
Need to remove the margin right for the last star
image

**Review by AnatoliiMarushchak — COMMENTED** [review]
Fixed a bug, improved the code.

**Review by etojeDenys — APPROVED** [review]
lgtm

**Pull request closed without merging**